src/index.html

```diff
@@ -7,8 +7,91 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="icon" href="./images/logo.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#">
+        <img 
+          src="./images/logo.png" 
+          alt="logo"
+        >
+      </a>
+      
+
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="apple"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link" href="samsung"
+            >
```

Comment:
```suggestion
            <a class="nav__link" href="samsung">
```

Reply:
you can leave attributes in one line if their count less then 3. but if you place them on a new line, use new line for each attribute